CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.11.0 Release
+- Release Date: August 16, 2018
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights
+
+#### Searching Archived Channels
+- Added ability to search for archived channels on desktop and mobile clients.
+
+#### Deep Linking
+- 
+
+### Improvements
+- Added profile pop-up to combined system messages.
+- Updated the home screen icon for Android.
+
+### Bug Fixes
+- Fixed an issue where Android showed the wrong time when the timezone setting was disabled.
+- Fixed an issue on Android where the app did not get sent to the background when pressing the hardware back button in the channel screen.
+- Fixed an issue on Android where pressing a push notification to reply to it sometimes went to the wrong channel.
+- Fixed an issue where the previous server URL was present on the input textbox before changing the screen to Login.
+- Fixed an issue where user menu was not translated after App was translated.
+- Fixed an issue where maximum field lengths didn't match web app when editing Account Settings.
+- Fixed an issue where long URLs for embedded images in message attachments got cut off and didn't render.
+- Fixed an issue where link preview images were not cropped properly.
+- Fixed an issue where long usernames didn't wrap properly in the Account Settings menu.
+- Fixed an issue where DMs would not open if users were using Jump To.
+- Fixed an issue where no message was displayed after removing a user from a channel with join/leave messages disabled.
+- Fixed an issue where Android showed the wrong time when timezone setting was disabled .
+
```

Comment:
* Fix an issue with Video playback when the filename had spaces
* Fix a crash when playing YouTube Videos
* Fix session expiration notification
* Fix sharing files from Google Drive in Android Share Extension